developer-guide/concepts-and-designs/sig-devops/devops_observability.md

```diff
@@ -0,0 +1,22 @@
+## Expose Pipiline metrics to Prometheus
+
+​	In order to have a more intuitive statistically view about pipelines. we should collect some metrics about pipelines, and then expose an endpoint with metrics where a Prometheus Server can scrape.
+
+​    Now we recommend using  [Jenkins Prometheus Plugin](https://github.com/jenkinsci/prometheus-plugin) to expose metrics about pipelines. Currently only metrics from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) and summary of build duration of jobs and pipeline stages.
+
+I picked-up some metrics which can be used in kubesphere.
+
+| Metrics                                                      | Type      | Description                                           | Note |
+| ------------------------------------------------------------ | --------- | ----------------------------------------------------- | ---- |
+| `jenkins_builds_duration_milliseconds_summary`               | `summary` | Summary of Jenkins build times in milliseconds by Job | ms   |
+| `jenkins_builds_success_build_count`                         | `counter` | Successful build count                                |      |
+| `jenkins_builds_failed_build_count`                          | `counter` | Failed build count                                    |      |
+| `jenkins_builds_last_build_result`                           | `gauge`   | Build status of a job as a boolean                    |      |
+| `jenkins_builds_last_build_duration_milliseconds`            | `gauge`   | Build times in milliseconds of last build             | ms   |
+| `jenkins_builds_last_build_start_time_milliseconds`          | `gauge`   | Last build start timestamp in milliseconds            | ms   |
+| `kubesphere_jenkins_builds_stage_duration_milliseconds_summary` | `summary` | Summary of Jenkins build times by Job and Stage       | ms   |
+| `jenkins_builds_last_build_result_ordinal`                   | ``        |                                                       |      |
+
+more info please to see **https://github.com/jenkinsci/metrics-plugin#metrics-plugin-for-jenkins**
```

Comment:
Do users need to integrate this plugin manually?

Reply:
no, we will install it default